From 562f7a73c2e5ece0c0cfdcfc7eb977c8e2e4a125 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 13 Jun 2018 09:45:57 +0530
Subject: [PATCH 0357/1851] v4l: xilinx: sdirxss: Correct the height of SD 525i
 to 486

commit 0902cba790b11e2ca35c8bbe3111082c868b069f from
https://github.com/Xilinx/linux-xlnx.git

Correct the height of SD 525i NTSC resolution from 480 to 486.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-sdirxss.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-sdirxss.c b/drivers/media/platform/xilinx/xilinx-sdirxss.c
index 1d360d1592b8..5d22324ef950 100644
--- a/drivers/media/platform/xilinx/xilinx-sdirxss.c
+++ b/drivers/media/platform/xilinx/xilinx-sdirxss.c
@@ -253,7 +253,7 @@
  * @XSDIRX_SMPTE_ST_296: SMPTE ST 296 detected with AP 1280x720
  * @XSDIRX_SMPTE_ST_2048_2: SMPTE ST 2048-2 detected with AP 2048x1080
  * @XSDIRX_SMPTE_ST_295: SMPTE ST 295 detected with AP 1920x1080
- * @XSDIRX_NTSC: NTSC encoding detected with AP 720x480
+ * @XSDIRX_NTSC: NTSC encoding detected with AP 720x486
  * @XSDIRX_PAL: PAL encoding detected with AP 720x576
  * @XSDIRX_TS_UNKNOWN: Unknown SMPTE Transport family type
  */
@@ -725,7 +725,7 @@ static int xsdirx_get_stream_properties(struct xsdirxss_state *state)
 		switch (family) {
 		case XSDIRX_NTSC:
 			format->width = 720;
-			format->height = 480;
+			format->height = 486;
 			break;
 		case XSDIRX_PAL:
 			format->width = 720;
-- 
2.31.1

